examples/main.tf

```diff
@@ -3,19 +3,19 @@
 terraform {
   required_providers {
     instaclustr = {
-      source = "instaclustr/instaclustr"
+      //source = "instaclustr/instaclustr"
```

Comment:
I'd suggest reverting your changes here, all users who download this will want to run it against production systems, not local testing